src/styles/components/card/_card-body.scss

```diff
@@ -1,10 +1,13 @@
+@use 'sass:color';
+
 .mynah-card-body {
   flex-shrink: 0;
   overflow: hidden;
   position: relative;
   display: block;
   line-height: var(--mynah-line-height);
   font-size: var(--mynah-font-size-medium);
+  word-break: break-all;
```

Comment:
This is pretty wrong here, what kind of a reason we're trying to break the words from middle inside a chat body?